packages/renderer/src/components/organisms/MarketsOffers/_utils.ts

```diff
@@ -0,0 +1,36 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+import { MarketOfferPaymentMethod } from "@molecules/MarketOffersTable";
+import type { MarketOfferData } from "@hooks/haveno/useMarketsOffers";
+import type { MarketOffer } from "@molecules/MarketOffersTable";
+
+export const transformToMarketsOffers = (
+  offers: MarketOfferData[]
+): MarketOffer[] => {
+  return offers.map((offer) => ({
+    price: offer.price,
+    priceCurrency: offer.counterCurrencyCode,
+    amount: offer.amount,
+    amountCurrency: offer.baseCurrencyCode,
+    costCurrency: offer.baseCurrencyCode,
+    priceComparison: 0.1,
```

Comment:
Are these constants: `priceComparison`, `cost`?

Reply:
Ok mapped the cost to `txFees` but we haven't price comparison or any percentage value.